lib/new_relic/agent/distributed_tracing/cross_app_tracing.rb

```diff
@@ -249,6 +251,10 @@ def assign_intrinsic_transaction_attributes(state)
           txn.attributes.add_intrinsic_attribute(:referring_transaction_guid, referring_guid)
         end
       end
+
+      def grpc_headers?(headers)
+        headers&.class&.name&.include?('::GRPC::')
```

Comment:
I love seeing these safe navigation operators!

Reply:
Removed with the [26e4415](https://github.com/newrelic/newrelic-ruby-agent/pull/1837/commits/26e4415437bb1e86f455b45bedc947507a7b0b93) rework.